### PR TITLE
docs(readme): Update README.md

### PR DESCRIPTION
**Pull request by mffap**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,39 @@
...
+## A Better Identity and Access Management Solution
+
+ZITADEL combines the best of Auth0 and Keycloak.
+It is built for the serverless era.
+
+Learn more about ZITADEL by checking out the [source repository on GitHub](https://github.com/zitadel/zitadel)
+
+## About this Terraform Provider
+
+This is the official ZITADEL terraform provider, which handles the migration part between different ZITADEL instances and different ZITADEL versions in addition to the creation of different resources with the help of the ZITADEL API.
```
Comment (eliobischof) [inline README.md:12]:
```suggestion
This is the official ZITADEL Terraform provider.
It lets you declare ZITADEL resources declaratively and apply the resources to any ZITADEL instance.
Importing existing resources to the Terraform state is supported.
Therefore, as long as you have all resources declared, it is even easy to migrate state between instances.
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,39 @@
...
+[Follow the Guide in our Docs](https://zitadel.com/docs/guides/manage/terraform/basics).
+Note that you need to create an authorized service user to access the ZITADEL APIs through the provider, as noted in the prerequisites.
+
+We don't guarantee that all resources are available in the provider.
+In case you miss something you are welcome to [contribute](#contributing).
+
+## Contributing
+
+If you found a bug or want to request a new feature, please open an [issue](https://github.com/zitadel/terraform-provider-zitadel/issues).
+Contributions to the provider are very welcome, please follow the general guidance in ZITADEL's [Contribution Guide](https://github.com/zitadel/zitadel/blob/main/CONTRIBUTING.md).
```
Comment (eliobischof) [inline README.md:33]:
```suggestion
Contributions to the provider are very welcome, please follow the general guidance in the [Contribution Guide](https://github.com/zitadel/terraform-provider-zitadel/blob/main/CONTRIBUTING.md).
```

Depends on https://github.com/zitadel/terraform-provider-zitadel/pull/96

**Review by eliobischof — DISMISSED** [review]
@stebenz we depend on https://github.com/zitadel/terraform-provider-zitadel/pull/96 here

File: README.md (hunk 2 above)
Comment (mffap) [reply README.md:33]: Yes, makes sense.

**Review by stebenz — APPROVED** [review]

**Pull request merged**